libraries/plugins/vfio/opae_vfio.c

```diff
@@ -762,6 +763,13 @@ fpga_result __VFIO_API__ vfio_fpgaClose(fpga_handle handle)
 		OPAE_ERR("invalid token in handle");
 	}
 
+	if (h->sva_fd) {
+		// Release PASID and shared virtual addressing
+		opae_close(h->sva_fd);
+		h->sva_fd = 0;
```

Comment:
0 is actually a valid file descriptor.  Could we use -1 as the closed sentinel for this?